puppetry/webcam/putils.py

```diff
@@ -38,10 +62,7 @@ def make_zxy_effector( name, point, output ):
                           frame
     '''
     pos = [ float( -point[2] ), float( point[0] ), float( -point[1] )  ]
-    if name in output:
-        output[name]['pos'] = pos
-    else:
-        output[name] = { 'pos' : pos }
+    add_ik(name,"pos", pos, output) 
```

Comment:
Whoops, I almost forgot: "pos" and "rot" are no longer accepted.  Need to be "p" or "position" and "r" or "rotation".